src/OpenTelemetry.Api/.publicApi/net461/PublicAPI.Shipped.txt

```diff
@@ -108,6 +108,7 @@ OpenTelemetry.Trace.TelemetrySpan.Dispose() -> void
 OpenTelemetry.Trace.TelemetrySpan.End() -> void
 OpenTelemetry.Trace.TelemetrySpan.End(System.DateTimeOffset endTimestamp) -> void
 OpenTelemetry.Trace.TelemetrySpan.IsRecording.get -> bool
+OpenTelemetry.Trace.TelemetrySpan.ParentId.get -> System.Diagnostics.ActivitySpanId
```

Comment:
all public changes should go to Unshipped.txt
(It'll be moved to shipped, once we ship a stable version with it)

Reply:
Ah ok, didn't know

Reply:
I'll fix this tonight

Reply:
Done